VikingQuiz.UI/src/Components/MyQuizesPage/MyQuizesPage.css

```diff
@@ -0,0 +1,31 @@
+.row{
+    margin-right:0;
+    margin-left: 0;
+}
+
+.quiz_Page_Container{
+    min-width: 320px;
+}
+
+.my_Quizzes_Page_Container{
+    margin: 0 7%;
+    margin-top: 40px;
+}
+
+.my_Profile_Container{
+    display: flex;
+    margin-right: 90%;
+}
+
+.new_Quiz{
+    margin-left:auto;
```

Comment:
Please format the code. There should be a space between the column ( ':' ) and the value for the property.